helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -1077,6 +1089,14 @@ alfresco-identity-adapter-service:
     - name: ACTIVITI_KEYCLOAK_CLIENT_SECRET
       value: "{{ tpl . $ }}"
     {{- end }}
+    {{- if .Values.global.hxpr.enabled }}
```

Comment:
Shouldn't we delete all "ifs" relalated hxpr?
If we right now have helmcharts are directly in https://github.com/HylandSoftware/terraform-aws-hxps-environment